package.json

```diff
@@ -74,7 +74,7 @@
     "turbo-test-runner": "0.3.3"
   },
   "engines": {
-    "node": ">=4.4"
+    "node": "4.4.x"
```

Comment:
The project works with versions upper than 4.4, so it should not be changed for now.